front/components/assistant_builder/types.ts

```diff
@@ -108,11 +107,11 @@ export type AssistantBuilderTableConfiguration =
 // Process configuration
 
 export type AssistantBuilderProcessConfiguration = {
-  timeFrame: AssistantBuilderTimeFrame;
+  timeFrame?: AssistantBuilderTimeFrame | null;
 } & {
   dataSourceConfigurations: DataSourceViewSelectionConfigurations;
   tagsFilter: AssistantBuilderTagsFilter | null;
-  schema: ProcessSchemaPropertyType[];
+  jsonSchema: string | null;
```

Comment:
We have a type for json schema I believe can't we use it whenever possible?

Introduced by @aubin-tchoi and @Fraggle I believe.